src/ima.js

```diff
@@ -454,6 +455,20 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     });
   }
 
+  /**
+   * set default Configuration for plugin
+   * @private
+   * @returns {void}
+   * @instance
+   * @memberof Ima
+   * @param {Object} config - configuration of app for ima plugin
+   */
+  _setDefaultConfiguration(config: Object): void {
```

Comment:
why do you need config if you are using `this` to access it anyway, seems redundant.

Reply:
It's not, in super method there is merge of all configurations.
I'm looking for the application configuration, this.config is the configuration after merge with default configuration so it always be false after merge with default(false).
So I'm taking the application config and check there assignment.